references/gcp-sa-auth-shared-flow/pipeline.sh

```diff
@@ -34,9 +34,9 @@ deleteKVM() {
 #clean up if the KVM and create cache if not already exists
 deleteKVM 'gcp-sa-devrel'
 
-curl -XPOST -u "$APIGEE_USER:$APIGEE_PASS" "https://api.enterprise.apigee.com/v1/o/$APIGEE_ORG/e/$APIGEE_ENV/caches" \
+curl -XPOST -s -u "$APIGEE_USER:$APIGEE_PASS" "https://api.enterprise.apigee.com/v1/o/$APIGEE_ORG/e/$APIGEE_ENV/caches" \
   -H 'Content-Type: application/json; charset=utf-8' \
-  --data-binary @- << EOF
+  --data-binary @- > /dev/null << EOF
```

Comment:
`-s` hides the progress meter and error messages but still shows the payload
`> /dev/null` also hides the response payload

is this what you expect? if so, please ignore this comment

Reply:
yeah just to keep the log output small